cyber-threat-intelligence/examples/06-uses-chain.tql

```diff
@@ -0,0 +1,23 @@
+# Example 06:
+
+# We will use the stix_id to find the attack pattern using the malware created in the Exercise 03
+# We will then create a uses relationship between the attack pattern and malware
+
+match
+$bravo_ap isa attack_pattern, has stix_id "attack_pattern_bravo";
+$bravo_ma isa malware, has stix_id "bravo_malware";
+
+insert
+$u1 (used: $bravo_ap, using:$bravo_ma) isa uses,
+    has stix_id "bravo_uses_1";
+
+#--------------------------------------------------
+# Exercise 06:
+
+# With inference switched on, we can now query for the attack pattern and malware that are used by the threat actor
+# Create a query that finds the attack pattern and malware that are used by the threat actor
+# You can double click on the green infered uses relationship to get an explanation of what triggered the inferred rule.
```

Comment:
You should probably mention that snapshot and explain need to be switched on for this to work.